### PR TITLE
Try to fix the ng e2e test #1436

### LINKED ISSUE #1436: "ng e2e --environment test" is test nothing after upgrading the Angular version
See this https://travis-ci.org/open-learning-exchange/planet/jobs/395550934#L891

After upgrading the Angular version seems karma test doesn't work.

screen shot 2018-06-23 at 12 09 30 am

### PR DESCRIPTION
**Pull request by mappuji**
Try to fix #1436

### CODE REVIEW

File: e2e/login.e2e-spec.ts [hunk 1]
```diff
@@ -22,7 +22,7 @@ describe('Login', () => {
     passInput.sendKeys('e2e');
     page.clickSignin();
     browser.getCurrentUrl().then((url) => {
-      expect(url).toEqual('http://0.0.0.0:49152/');
+      expect(url).toEqual('http://localhost:4200/');
```
Comment (mappuji) [inline e2e/login.e2e-spec.ts]: can url stored in config?

Comment (paulbert) [reply e2e/login.e2e-spec.ts]: When we set up more tests, yes this should be set in config.

**Comment by paulbert** [comment]
Found a possible problem:  For the latest commits the first two `planet-test` builds are not being tagged with the branch name and commit id:

image

**Comment by mappuji** [comment]
Thanks @paulbert will take a look at it.

**Pull request merged**